components/elm/docs/tech-guide/shortwave_radiation.md

```diff
@@ -0,0 +1,812 @@
+  albedo.
+- The model accounts for radiation in two bands: visible
+  ($< 0.7\mu$m) and near-infrared ($\ge 0.7 \mu$m).
+- The model uses the two-stream approximation of
+  Dickenson 1983[@dickinson1983land] and
+  Sellers 1985[@sellers1985canopy].
+- The model is a single-layer canopy model.
+- Sunlit leaf absorbs unscattered direct radiation,
+  scattered direct radiation, and diffuse radiation.
+- Shaded leaf absorbs scattered direct radiation and
+  diffuse radiation.
+- Snow albedo and the shortwave radiative heating of each snowpack layer is modeled
+  is modeled using the hybrid SNICAR-AD model
+  of Dang et al. 2019[@dang2019intercomparison].
+- The sub-grid impacts of topography on solar radiation
+  are parameterized using the approach of Hao et al. 2021
+  [@hao2021parameterization].
+
+ELM's shortwave radiation model is similar to
+CLM4.5[@oleson2013clm45], and its new features include
+the hybrid SNICAR-AD model for snow albedo and a
+parameterization for the sub-grid effects of topography
+on solar radiation.
+
+The outline of this chapter is as follows. First, the
+governing equations of the two-stream approximation of
+the radiative transfer model are presented. Second, the analytical
+solutions for the upward and downward diffuse radiation
+using the two-stream approximation are given. Third, the
+analytical solutions are used for the single-layer canopy
+configuration to compute the radiation absorbed by the
+canopy and ground and the radiation absorbed by the
+canopy for sunlit and shaded leaves. Fourth, the
+treatment of surface albedo is described. Fifth, the
+equations describing the total amount of solar radiation
+are presented.
+
+## Two-Stream Approximation Radiative Transfer Model
+
+At any level within the canopy, the upward/downward
+direct/diffuse radiation is transmitted and intercepted
+by the canopy. The intercepted radiation is scattered
+backward, scattered forward, and absorbed by the canopy.
+The diffuse radiative fluxes via the two-stream
+approximation are given as
+
+$$
+\begin{eqnarray}
+- \dfrac{dI^{\uparrow}}{d(L+S)} + [ 1 - (1 - \beta)\omega ]
+\dfrac{1}{\bar{\mu}}I^{\uparrow} - \omega \beta \dfrac{1}{\bar{\mu}}
+I^{\downarrow} = \omega K_b \beta_0 e^{-K(L+S)}
+\label{eqn:dif_up} \\[0.5em]
+ \dfrac{dI^{\downarrow}}{d(L+S)} + [ 1 - (1 - \beta)\omega ]
+\dfrac{1}{\bar{\mu}} I^{\downarrow} - \omega \beta \dfrac{1}{\bar{\mu}}
+I^{\uparrow} = \omega K_b (1 - \beta_0) e^{-K(L+S)}
+\label{eqn:dif_dn}
+\end{eqnarray}
+$$
+
+where $I^{\uparrow}$ is the upward diffuse radiative flux
+per unit incident flux, $I^{\downarrow}$ is the downward
+diffuse radiative flux per unit incident flux,
+$K_b \left( = G(\mu)/\mu \right)$ is the optical depth of
+direct radiation per unit leaf and stem area, $G(\mu)$ is
+the relative projected area of leaf and stem in the
+direction of $\cos^{-1}(\mu)$, $\mu$ is the solar zenith
+angle, $\bar{\mu} \left( = 1/K_d \right)$ is the inverse
```

Comment:
"in the direction of $\cos^{-1}(\mu)$, $\mu$ is the solar zenith
angle" I think this should be "in the direction of $\mu$, the solar zenith
angle". The inverse cosine function returns an angle when given a ratio as input, but as written you are providing it with an angle as input.

Reply:
@thorntonpe Be careful. I think \mu is the cosine of the solar zenith angle (usually denoted \theta), so that "in the
direction of $\cos^{-1}(\mu)$" is correct but the following phrase should say "$\mu$ is the cosine of the solar zenith
angle,"

Reply:
@czender is correct that the description of $\mu$ should state that it is the cosine of the solar zenith angle. I will fix it.